src/main/controllers/RemoveListSearchResultsController.ts

```diff
@@ -6,20 +6,35 @@ import { SummaryOfPublicationsService } from '../service/SummaryOfPublicationsSe
 import { ManualUploadService } from '../service/ManualUploadService';
 
 const courtService = new LocationService();
-const publicationService = new SummaryOfPublicationsService();
+const summaryOfPublicationsService = new SummaryOfPublicationsService();
 const manualUploadService = new ManualUploadService();
 
 export default class RemoveListSearchResultsController {
     public async get(req: PipRequest, res: Response): Promise<void> {
         const locationId = parseInt(req.query?.locationId as string);
+        const noOptionSelectedError = req.query?.error;
         locationId
             ? res.render('remove-list-search-results', {
                   ...cloneDeep(req.i18n.getDataByLanguage(req.lng)['remove-list-search-results']),
                   court: await courtService.getLocationById(locationId),
                   removalList: manualUploadService.formatListRemovalValues(
-                      await publicationService.getPublications(locationId, req.user?.['userId'], true)
+                      await summaryOfPublicationsService.getPublications(locationId, req.user?.['userId'], true)
                   ),
+                  noOptionSelectedError: noOptionSelectedError,
               })
             : res.render('error', req.i18n.getDataByLanguage(req.lng).error);
     }
+
+    public async post(req: PipRequest, res: Response): Promise<void> {
+        if (req.user) {
+            if (req.body.courtLists == undefined) {
```

Comment:
I would add a '?' after the body. Very unlikely for it to be sent with no body, but I think it's worth capturing:

'req.body?.courtLists'

I'm also not sure you need the == undefined - I think 

if (req.body?.courtLists) 

will work